.gitmodules

```diff
@@ -43,10 +43,6 @@
   path = 3party/googletest
   url = https://github.com/google/googletest.git
   branch = main
-[submodule "3party/fast_double_parser"]
-  path = 3party/fast_double_parser
-  url = https://github.com/lemire/fast_double_parser.git
```

Comment:
There is from_chars implementation from the same guy that can be used on all platforms (it would be great to compare it with fast_double_parser first):
https://github.com/fastfloat/fast_float